modules/govuk/manifests/apps/email_alert_api/delivery_attempt_status_update_check.pp

```diff
@@ -0,0 +1,26 @@
+# == Define: govuk::apps::email_alert_api::delivery_attempt_status_update_check
+#
+# Creates an Icinga check which checks Graphite for pending delivery attempts.
+#
+# === Parameters
+#
+# [*ensure*]
+#   Whether to enable the check.
+#   Default: present
+#
+# [*title*]
+#   The status of the delivery attempts to look for.
+#
+define govuk::apps::email_alert_api::delivery_attempt_status_update_check(
+  $ensure = present,
+) {
+  @@icinga::check::graphite { "email-alert-api-delivery-attempt-${title}":
```

Comment:
I think this definition should go in https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk/manifests/apps/email_alert_api/checks.pp (which is included from here https://github.com/alphagov/govuk-puppet/blob/de2adb57d7d6b347c10195c54b34727987759af0/modules/monitoring/manifests/checks.pp#L50) as currently nothing is using `govuk::apps::email_alert_api::delivery_attempt_status_update_check` so it won't be included anywhere.